boards/wio_lite_w600/examples/blinky_basic.rs

```diff
@@ -0,0 +1,40 @@
+#![no_std]
+#![no_main]
+
+#[cfg(not(feature = "use_semihosting"))]
+use panic_halt as _;
+
+use wio_lite_w600 as bsp;
+
+use bsp::entry;
+use bsp::hal;
+
+use hal::clock::GenericClockController;
+use hal::delay::Delay;
+use hal::pac::{CorePeripherals, Peripherals};
+use hal::prelude::*;
+
+#[entry]
+fn main() -> ! {
+    let mut peripherals = Peripherals::take().unwrap();
+    let core = CorePeripherals::take().unwrap();
+    let mut clocks = GenericClockController::with_external_32kosc(
+        peripherals.GCLK,
+        &mut peripherals.PM,
+        &mut peripherals.SYSCTRL,
+        &mut peripherals.NVMCTRL,
+    );
+    let pins = hal::Pins::new(peripherals.PORT);
```

Comment:
I think this should be `bsp::Pins::new(peripherals.PORT)`

Reply:
Whoops! Fixed. Thanks for that. 😅